ctapipe/image/pixel_likelihood.py

```diff
@@ -91,9 +91,9 @@ def neg_log_likelihood_approx(image, prediction, spe_width, pedestal):
     -------
     float
     """
-    theta = pedestal ** 2 + prediction * (1 + spe_width ** 2)
+    theta = pedestal**2 + prediction * (1 + spe_width**2)
 
-    neg_log_l = np.log(theta) + (image - prediction) ** 2 / theta
+    neg_log_l = np.log(theta + 1e-323) + (image - prediction) ** 2 / theta
```

Comment:
I would suggest defining `EPSILON = 1.0e-323` at the top of the file and avoid a bare number here (and also below).